app/index.html

```diff
@@ -3,7 +3,6 @@
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width,initial-scale=1">
 <title>HiGlass</title>
-<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
Was bootstrap finally removed? If so might be worth mentioning in the changelog

Reply:
Yeah, it's gone. It's been gone for a while but this stylesheet link wasn't removed.

I'm not going to add it to the CHANGELOG because it would imply that bootstrap was removed in this PR when it wasn't.